pinc/ProjectSearchResults.inc

```diff
@@ -622,7 +622,7 @@ class ProjectSearchResults
         if(!empty($warning))
             echo "\n<p class='warning'>$warning</p>\n";
 
-        echo "<p id='head'>" . _("To set primary sort term, click on the column name; to set secondary sort term, click on the <u>^</u> that follows the column name.") . "</p>";
+        echo "<p id='head'>" . _("To set primary sort term, click on the column name; to set secondary sort term, click on the <span style='text-decoration: underline'>^</span> that follows the column name.") . "</p>";
```

Comment:
If the `text-decoration: underline` is likely to be needed in multiple places, shouldn't it be defined in the site CSS somewhere?

Reply:
That's possible, though I wasn't sure the standard for using css vs inline style. I see inline styles frequently so followed that approach.

Reply:
We actually have a class for underline already as one of our mixins. You should be able to use:
```
<span class='underlined'>
```

Reply:
Nice, I missed that one somehow. I updated to use it here and in some other places. I hope it is ok.